src/main/java/ru/sevsu/rc_manager/sound/handler/Handler.java

```diff
@@ -1,7 +1,9 @@
 package ru.sevsu.rc_manager.sound.handler;
 
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
 import javax.sound.sampled.AudioInputStream;
 
 public interface Handler {
-    void handle(AudioInputStream audioInputStream);
+    void handle(AudioInputStream audioInputStream) throws TelegramApiException;
```

Comment:
In general we can just wrap it with SneakyThrows lombok annotation instead of adding `throws` for every exception